hnn_core/network.py

```diff
@@ -486,8 +488,16 @@ def add_evoked_drive(self, name, *, mu, sigma, numspikes, location,
         space_constant : float
             Describes lateral dispersion (from column origin) of synaptic
             weights and delays within the simulated column
-        seedcore : int
+        probability : dict or float (default: 1.0)
+            Probability of connection between any src-target pair.
+            Use dict to create probability->cell mapping. If float, applies to
+            all target cell types
+        event_seed : int
             Optional initial seed for random number generator (default: 2).
+            Used to generate event times for drive cells.
+        conn_seed : int
+            Optional initial seed for random number generator (default: 3).
+            Used to randomly remove connections when probablity < 1.0.
```

Comment:
I think there is some missing documentation here. Normally seed is `None` by default. Here you set it an integer so that the connections are consistent across trials? Would be good to spell out across what the connections are consistent

Reply:
Very good point, I honestly didn't have a motivation behind fixing the seed across trials but it is a pleasant (on unpleasant) side effect depending on the question you're interested in, I'll add a better description